app/assets/stylesheets/message_form.css

```diff
@@ -0,0 +1,3 @@
+/* #message{ */
+/*   background-color: blue; */
+/* } */
```

Comment:
Maybe you can use sass instead of pure css

Reply:
Yes indeed! :) , and actually that is commented because i was testing if the file was being loaded